router/throttler/adaptive.go

```diff
@@ -16,6 +16,8 @@ type adaptiveThrottleConfig struct {
 	maxLimit func() int64
 }
 
+const DefaultMaxLimit = 1000 // 1000 requests per second
```

Comment:
Does this need to be exported? It's being used within the package only?

Reply:
Made the change